index.html

```diff
@@ -0,0 +1,27 @@
+---
+Title: All of the CVs
```

Comment:
I remain convinced that the index page should be the default CV.
Many users fork the repo so that it points to theirawesomewebsiteurl.github.io/cv directly. 
If what you want is to show different 'profiles' to companies then you probably won't welcome a page that lists all of them in one place?

If you want to have links to the other versions, maybe they should be included at the top of the default cv?

Reply:
All of those are good points. 

> If you want to have links to the other versions, maybe they should be included at the top of the default cv?

Yeah, but that would still block users from easily printing their CV I guess. 

Given all of this, I think I will close the PR as I think we are trying to serve different users. Thanks a lot for reviewing anyway!